### PR TITLE
HAWQ-925. Set default locale, timezone & datastyle before running sql…

### PR DESCRIPTION
**Pull request by paul-guo-**
… command/file

### CODE REVIEW

File: src/test/feature/partition/ans/partition_negetive_and_basics.ans [hunk 1]
```diff
@@ -1089,7 +1085,7 @@ INSERT 0 1
 insert into ggg values (9, 9);
 INSERT 0 1
 insert into ggg values (10, 10);
-psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:566: ERROR:  no partition for partitioning key  (seg1 localhost:40000 pid=9097)
+psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:566: ERROR:  no partition for partitioning key  (seg5 localhost:40000 pid=172591)
```
Comment (xunzhang) [inline src/test/feature/partition/ans/partition_negetive_and_basics.ans]: I think we should ignore this line?

File: src/test/feature/partition/ans/partition_negetive_and_basics.ans [hunk 2]
```diff
@@ -1321,7 +1317,7 @@ psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:690: NOTICE:  CREATE
 ALTER TABLE
 -- works - anonymous partition MPP-3350
 alter table hhh add partition end ('2010-02-01');
-psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:693: NOTICE:  CREATE TABLE will create partition "hhh_1_prt_r418565524" for table "hhh"
+psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:693: NOTICE:  CREATE TABLE will create partition "hhh_1_prt_r652516349" for table "hhh"
```
Comment (xunzhang) [inline src/test/feature/partition/ans/partition_negetive_and_basics.ans]: I think we should ignore this line too.

File: src/test/feature/partition/ans/partition_negetive_and_basics.ans [hunk 3]
```diff
@@ -1589,7 +1585,7 @@ insert into j select i from generate_series(1, 8) i;
 INSERT 0 8
 alter table j split partition for(1) at (2, 3) into (partition fa, partition
 fb);
-psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:870: NOTICE:  exchanged partition "a" of relation "j" with relation "pg_temp_54636"
+psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:870: NOTICE:  exchanged partition "a" of relation "j" with relation "pg_temp_43517"
```
Comment (xunzhang) [inline src/test/feature/partition/ans/partition_negetive_and_basics.ans]: ditto

File: src/test/feature/partition/ans/partition_negetive_and_basics.ans [hunk 4]
```diff
@@ -1609,10 +1605,10 @@ select * from j_1_prt_fb;
 (2 rows)
 
 alter table j split partition for(5) at (6);
-psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  exchanged partition "b" of relation "j" with relation "pg_temp_54636"
+psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  exchanged partition "b" of relation "j" with relation "pg_temp_43517"
```
Comment (xunzhang) [inline src/test/feature/partition/ans/partition_negetive_and_basics.ans]: ditto

File: src/test/feature/partition/ans/partition_negetive_and_basics.ans [hunk 5]
```diff
@@ -1609,10 +1605,10 @@ select * from j_1_prt_fb;
...
 psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  dropped partition "b" for relation "j"
-psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  CREATE TABLE will create partition "j_1_prt_r613391202" for table "j"
-psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  CREATE TABLE will create partition "j_1_prt_r1064781686" for table "j"
+psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  CREATE TABLE will create partition "j_1_prt_r630513650" for table "j"
+psql:/tmp/TestPartition_TestPartitionNegativeAndBasics.sql:873: NOTICE:  CREATE TABLE will create partition "j_1_prt_r1373370768" for table "j"
```
Comment (xunzhang) [inline src/test/feature/partition/ans/partition_negetive_and_basics.ans]: ditto

**Comment by zhangh43** [comment]
+1

File: src/test/feature/partition/ans/partition_negetive_and_basics.ans (hunk 1 above)
Comment (paul-guo-) [inline src/test/feature/partition/ans/partition_negetive_and_basics.ans]: This difference seems to be eliminated by atmsort.pl.

**Comment by xunzhang** [comment]
+1

**Pull request merged**